org.gabmus.unifydmin.json

```diff
@@ -0,0 +1,410 @@
+            "buildsystem": "simple",
+            "build-options": {
+                "arch": {
+                    "i386": {
+                        "env": {
+                            "MAX_CONCURRENCY": "1"
+                        }
+                    },
+                    "arm": {
+                        "env": {
+                            "MAX_CONCURRENCY": "1"
+                        }
+                    }
+                }
+            },
+            "ensure-writable": [
+                "/lib/python*/site-packages/easy-install.pth",
+                "/lib/python*/site-packages/setuptools.pth",
+                "/app/lib/python*/site-packages/easy-install.pth",
+                "/app/lib/python*/site-packages/setuptools.pth"
+            ],
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/micheles/decorator/archive/4.3.2.tar.gz",
+                "sha256": "c053ec881270608a5aef624a45a7700ec76ceca9d96689506411e1f319085380"
+            }]
+        },
+        {
+            "name": "python-fabric",
+            "buildsystem": "simple",
+            "build-options": {
+                "arch": {
+                    "i386": {
+                        "env": {
+                            "MAX_CONCURRENCY": "1"
+                        }
+                    },
+                    "arm": {
+                        "env": {
+                            "MAX_CONCURRENCY": "1"
+                        }
+                    }
+                }
+            },
+            "ensure-writable": [
+                "/lib/python*/site-packages/easy-install.pth",
+                "/lib/python*/site-packages/setuptools.pth",
+                "/app/lib/python*/site-packages/easy-install.pth",
+                "/app/lib/python*/site-packages/setuptools.pth"
+            ],
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/fabric/fabric/archive/2.4.0.tar.gz",
+                "sha256": "cb562fe648c8a473be8684a5a36494c0b8d1d2b3654776f1628df2c0808f236b"
+            }]
+        },
+        "libdazzle.json",
+        {
+            "name": "libhandy",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": ["--buildtype=release"],
+            "sources": [{
+                "type": "archive",
+                "url": "https://source.puri.sm/Librem5/libhandy/-/archive/v0.0.9/libhandy-v0.0.9.tar.gz",
+                "sha256": "a9f3c06eaa482efabc1d00dd2df65dc7d41253a71c0cb0402328a831a4f398a3"
+            }]
+        },
+        {
+            "name": "unifydmin",
+            "buildsystem": "meson",
+            "builddir": true,
```

Comment:
Not needed.

Reply:
do you mean `"builddir": true,` is not needed?

Reply:
@GabMus 
> do you mean `"builddir": true,` is not needed?

It is default for the `meson` buildsystem.

Reply:
done